src/%IPM/Main.cls

```diff
@@ -156,75 +188,111 @@ repo -n LocalFiles -delete
 <example description="Lists all modules (and versions) available from the repository named &quot;AppModules&quot;">
 repo -n AppModules -list-modules
 </example>
+<example description="Enables package manager web services in this namespace via the /csp/mynamespace/ web application, including web services for publishing. Note that security (e.g., authentication settings, required role) should be configured separately as part of this web application.">
+repo -enable -app /csp/mynamespace -publish 1
+</example>
 
+<!-- Modifiers -->
 <!-- Universal Actions -->
-<modifier name="delete-all" description="Deletes all repositories (possibly subject to a type filter: -f, -r, ...)" />
-<modifier name="list" description="Lists all repositories (possibly subject to a type filter: -f, -r, ...)" />
+<modifier name="delete-all" description="Deletes all repositories (possibly subject to a type filter: -filesystem, -remote, -local, -perforce)" />
+<modifier name="list" description="Lists all repositories (possibly subject to a type filter: -filesystem, -remote, -local, -perforce)" />
 <modifier name="list-modules" aliases="modules,listmodules" description="List modules available in the specified repository (-n\[ame]), or in all configured repositories if no repository was specified." />
-
+<modifier name="enable" description="Enables package manager web services in this namespace." />
```

Comment:
Do all of these work in IPM? Should confirm before adding them to the help command

Reply:
@isc-kiyer  Have adjusted all the newly added ones:

* enable, disable, info, app - repo: Those are related to %IPM.PackageManager.Server.PKG, which we don't support so remove.
* local - repo: No support for local server application (app) so remove it
* branch - load: verified that it is supported
* latest - install: verified that it is supported
* manage(command): related to app so remove
* clear-cache(command): related to %IPM.PackageManager.Server.PKG (whose part manages application artifact) so remove
* showsource, repository - list-installed: we should support them. However, current version of CommunityPM has added very unreasonable code (see GetListModule() and GetListModules()) with bad comments to handle some of the list-installed work that I cannot immediately add their logic back. I have created https://usjira.iscinternal.com/browse/HSIEO-6370 to handle them.

Now all helps should match the actual functions